src/grammar/query.rs

```diff
@@ -22,10 +22,42 @@ pub(crate) fn parse_query(p: &mut Parser, expect_into_clause: bool) {
         parse_where_clause(p);
     }
 
+    if p.at(T![connect]) {
+        parse_connect_by(p);
+    }
+    if p.at(T![starts]) {
+        parse_starts_with(p);
+    }
+
```

Comment:
This technically allows you to write the following
```sql
...
      CONNECT BY PRIOR employee_id = manager_id
      START WITH employee_id = 100
      START WITH employee_id = 100
      CONNECT BY PRIOR employee_id = manager_id
...
```

What do you think about using a `match` together with `p.current()`?